internal/services/formatters/go/nancy/formatter.go

```diff
@@ -25,16 +25,16 @@ import (
 	"github.com/ZupIT/horusec-devkit/pkg/enums/tools"
 	"github.com/ZupIT/horusec-devkit/pkg/utils/logger"
 
-	dockerEntities "github.com/ZupIT/horusec/internal/entities/docker"
+	"github.com/ZupIT/horusec/internal/entities/docker"
 	"github.com/ZupIT/horusec/internal/enums/images"
 	"github.com/ZupIT/horusec/internal/helpers/messages"
 	"github.com/ZupIT/horusec/internal/services/formatters"
-	"github.com/ZupIT/horusec/internal/services/formatters/go/nancy/entities"
-	"github.com/ZupIT/horusec/internal/services/formatters/go/nancy/enums"
 	"github.com/ZupIT/horusec/internal/utils/file"
 	vulnHash "github.com/ZupIT/horusec/internal/utils/vuln_hash"
 )
 
+const goModulesExt = ".mod"
```

Comment:
should be in `internal/services/formatters/go/nancy/vulnerability.go` const declaration?

Reply:
Actually not. This const is used only on `internal/services/formatters/go/nancy/formatter.go`